requirements.txt

```diff
@@ -26,7 +26,9 @@ flask==1.0.2
 flask-cors==3.0.3
 flask_swagger==0.2.12
 forbiddenfruit==0.1.3
+jsonpatch==1.24
 jsonpath-rw==1.4.0
+jsonpointer==2.0
```

Comment:
Do we need this dependency? Looks like we're not importing it - is it required as a transitive dependency somewhere?

Reply:
is a dep of jsonpatch

Reply:
I see. It's also listed in the [`requirements.txt`](https://github.com/stefankoegl/python-json-patch/blob/master/requirements.txt), so technically not needed to import it here (unless we deliberately want to pin it to a specific version). But it's fine, shouldn't hold up the merge. thx